clients/spring-boot-starter-camunda-sdk/src/main/java/io/camunda/spring/client/configuration/ExecutorServiceConfiguration.java

```diff
@@ -51,7 +52,9 @@ public CamundaClientExecutorService camundaClientThreadPool(
     if (meterRegistry != null) {
       final MeterBinder threadPoolMetrics =
           new ExecutorServiceMetrics(
-              threadPool, "zeebe_client_thread_pool", Collections.emptyList());
+              threadPool,
+              "camundaClientExecutor",
```

Comment:
💭 I lack knowledge on the micrometer API here, are name tags a way to have an alias executorServiceName? Or will the intoduction of the new name tag require some changes to prometheus query/existing usages of this metric in Grafana?